Orange/widgets/visualize/owradviz.py

```diff
@@ -47,7 +46,7 @@ def __init__(self, master):
 
         self.master = master
         self.n_neighbors = 10
-        max_n_attrs = len(master.model_selected) + len(master.model_other) - 1
+        max_n_attrs = 0
```

Comment:
It may had been meant as just a temporary fix because the code crashed after eliminating `model_selected` and `model_other`. This however never worked, I suppose: Vizrank's `__init__` is called from widget's `__init__` not from `set_data`, so the number of attributes is unknown, and also not changed when the data changes.

I changed the max to 99. If the number of attributes is smaller, it's OK. If it's larger, this limit is sensible, too.